packages/gitmoji-changelog-markdown/src/index.spec.js

```diff
@@ -32,6 +54,54 @@ describe('Markdown converter', () => {
             },
           ],
         },
+      ],
+    }
+
+    await buildMarkdownFile(changelog, { mode: 'init', output: './CHANGELOG.md' })
+
+    expect(fs.writeFileSync.mock.calls[0].length).toBe(2)
+    expect(fs.writeFileSync.mock.calls[0][0]).toBe('./CHANGELOG.md')
+    expect(fs.writeFileSync.mock.calls[0][1]).toEqual(`# Changelog
+
+<a name="next"></a>
+## next
+
+### Changed
+
+- :recycle: Upgrade brand new feature ([c40ee86](https://github.com/frinyvonnick/gitmoji-changelog/commit/c40ee8669ba7ea5151adc2942fa8a7fc98d9e23c))
+
+
+<a name="1.0.0"></a>
+## 1.0.0 - 2018-08-28
+
+### Added
+
+- :sparkles: Upgrade brand new feature ([c40ee86](https://github.com/frinyvonnick/gitmoji-changelog/commit/c40ee8669ba7ea5151adc2942fa8a7fc98d9e23f))
+
+
+`)
+  })
+
+  // FIXME: should find a better way to mock fs
+  it.skip('should generate incremental markdown changelog', async () => {
```

Comment:
This test works but it is not full, I didn't succeed to mock streams of `fs`, it runs in conflict with `jest` that use `fs` internally. We should find a better way to mock `fs`